### PR TITLE
Update MHN theme for 4.8

### PR DESCRIPTION
**Pull request by ranjithsiji**
Update MHN theme variables to run the theme on phpMyAdmin 4.8

### CODE REVIEW

File: mhn/css/common.css.php [hunk 1]
```diff
@@ -25,7 +29,7 @@
 progress[value]::-moz-progress-bar{background-color:<?php echo $GLOBALS['cfg']['cPri']; ?>}
 fieldset{border:none}
 <?php require_once('mhn.icons.css.php'); ?>
-html{font-size:<?php echo $_SESSION['PMA_Theme']->getFontSize(); ?>}
+html{font-size:1em;}
```
Comment (mauriciofauth) [inline mhn/css/common.css.php]: 4.8 still supports `getFontSize`.

**Comment by ranjithsiji** [comment]
OK the code reinstated. now using getFontSize. And thank you for reviewing my request.

**Comment by ibennetch** [comment]
This looks great to me, @mauriciofauth are you satisfied with the changes?

**Comment by ranjithsiji** [comment]
Thanks for accepting my Request.

**Comment by ibennetch** [comment]
Squashed and merged as abcf3593a27a588bd8bb5feeb6df41c6ebfc78ab.

Released the new version 1.4 at https://www.phpmyadmin.net/themes/

Thank you for your contribution!

**Pull request merged**